src/webelement.rs

```diff
@@ -242,12 +244,54 @@ impl<'a> WebElement<'a> {
     }
 
     /// Get the specified property.
+    ///
+    /// # Example:
+    /// ```rust
+    /// # use thirtyfour::prelude::*;
+    /// # use thirtyfour::support::block_on;
+    /// #
+    /// # fn main() -> WebDriverResult<()> {
+    /// #     block_on(async {
+    /// #         let caps = DesiredCapabilities::chrome();
+    /// #         let driver = WebDriver::new("http://localhost:4444/wd/hub", &caps).await?;
+    /// #         driver.get("http://webappdemo").await?;
+    /// #         driver.find_element(By::Id("pagetextinput")).await?.click().await?;
+    /// #         let elem = driver.find_element(By::Name("input2")).await?;
+    /// #         //Convert property value to string because its type is boolean, get_property
+    /// #         let property_value = elem.get_property("checked").await?;
+    /// #         assert_eq!(property_value, "true");
+    /// #         Ok(())
+    /// #     })
+    /// # }
+    /// ```
     pub async fn get_property(&self, name: &str) -> WebDriverResult<String> {
         let v = self.cmd(Command::GetElementProperty(&self.element_id, name.to_owned())).await?;
+        if !is_string(&v["value"]) {
+           return convert_json(&json!(&v["value"].to_string()))
+        }
         convert_json(&v["value"])
```

Comment:
Is there a reason why we can't just do the following instead of using convert_json() ?

```
Ok((&v["value"].to_string()).clone())
```

What other values do you expect here other than string/boolean?

Reply:
No we don't. I believe `to_string()` already allocates anyway.